packages/valory/skills/market_manager_abci/bets.py

```diff
@@ -108,7 +108,16 @@ def __post_init__(self) -> None:
 
     def __lt__(self, other: "Bet") -> bool:
         """Implements less than operator."""
-        return self.scaledLiquidityMeasure < other.scaledLiquidityMeasure
+        outcome_index = self.prediction_response.vote
+
+        if self.n_bets > 0 and outcome_index:
```

Comment:
What if the vote is `0`?

Reply:
ah this needs to be outcome_index is not None